docs/docs/hive.md

```diff
@@ -91,20 +91,9 @@ Hive supports the following additional features with Hive version 4.0.0 and abov
 
 Hive 4 comes with `hive-iceberg` that ships Iceberg, so no additional downloads or jars are needed. For older versions of Hive a runtime jar has to be added.
 
-### Hive 4.0.0
+### Hive 4.0.x
 
-Hive 4.0.0 comes with the Iceberg 1.4.3 included.
-
-### Hive 4.0.0-beta-1
-
-Hive 4.0.0-beta-1 comes with the Iceberg 1.3.0 included.
-
-### Hive 4.0.0-alpha-2
-
-Hive 4.0.0-alpha-2 comes with the Iceberg 0.14.1 included.
-### Hive 4.0.0-alpha-1
-
-Hive 4.0.0-alpha-1 comes with the Iceberg 0.13.1 included.
+Hive 4.0.x comes with the Iceberg 1.4.3 included.
```

Comment:
- [4.0.0](https://github.com/apache/hive/blob/rel/release-4.0.0/iceberg/pom.xml#L28)
- [4.0.1](https://github.com/apache/hive/blob/rel/release-4.0.1/iceberg/pom.xml#L28)

Reply:
Nit: can we remove 'the' in front of Iceberg while we are here.  Its not common to have the article before Iceberg.